core/src/create.jl

```diff
@@ -189,15 +189,18 @@ function Control(db::DB, config::Config)::Control
     )
 end
 
-function PIDControl(db::DB, config::Config)::PIDControl
-    static = load_structvector(db, config, PIDControlStaticV1)
+function PidControl(db::DB, config::Config)::PidControl
```

Comment:
Why did you rename from PIDControl to PidControl? PID controllers are typically spelled with uppercase PID since it is an acronym.

Reply:
That is because of the automatic conversion between snake case and upper camel case.